go.mod

```diff
@@ -3,18 +3,18 @@ module github.com/planetscale/psdb
 go 1.22
 
 require (
-	connectrpc.com/connect v1.14.0
+	connectrpc.com/connect v1.18.1
```

Comment:
We can't upgrade this yet.

Reply:
I'll just undo the update and just do the changes.